app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -319,9 +267,42 @@ def validate_channel(self, value):
         return value
 
 
-class NimbusStatusUpdateSerializer(
-    NimbusChangeLogMixin, NimbusStatusRestrictionMixin, serializers.ModelSerializer
+class NimbusExperimentUpdateSerializer(
+    NimbusChangeLogMixin,
```

Comment:
Huh, I really need to read up / get refreshed on how multiple inheritance here works with REST Framework. Really nice that all of that can just be converted into mixins with `validate` methods!

Reply:
Huzzah!